scripts/dts/gen_defines.py

```diff
@@ -435,11 +435,23 @@ def irq_name_alias(irq, cell_name):
             alias += "_" + str2ident(cell_name)
         return alias
 
+    def encode_zephyr_multi_level_irq(irq, irq_num):
+        # See doc/reference/kernel/other/interrupts.rst for details
+        # on how this encoding works
```

Comment:
Blank line after doc comment, to just be consistent with elsewhere.

Does this code make any simplifying assumptions? For example, if some interrupt controller in turn sends interrupts to five other controllers (via `interrupts-extended`), then does the code make sense?

Should mention simplifying assumptions in a comment, if there are any.

Reply:
Can't think of any assumptions.  Should work fine w/regards to `interrupt-extended`  or `interrupt-maps`.

Reply:
Okay, fine then.